web/src/App.js

```diff
@@ -0,0 +1,55 @@
+import React, { Component } from 'react';
+import './styles/global.css';
+
+import Register from './components/register/register';
+import GroupView from './components/groupView/groupView';
+import Wave from './components/wave/wave';
+import './App.css';
+
+class App extends Component {
+  constructor(props) {
+    super(props)
+    
+    this.state = {
+      groupInfo: null
+    }
+    
+    this.handleUserData = this.handleUserData.bind(this)
+  }
+  
+  handleUserData(data) {
+    fetch("/group/"+data.groupId)
+      .then(response => response.json())
+      .then(res => {
+        if(res.message) {
+          alert("Unfortunately there was an error, please try again later.")
```

Comment:
Do we want to use alerts? Correct me if I am wrong but these are popup dialogs, I dislike encountering them as a user. Just an error message in the page would be better suited for this. I am open to hearing your thoughts.
cc @frzifus

Reply:
I am convinced that the user shouldn't see error messages at all, but in the unlikely event that an error should occur, an alert would be the easiest way. Normally I would prefer console output but in this case the user wouldn't get any feedback and waits in vain for an answer.